sunpy/timeseries/tests/test_timeseries_factory.py

```diff
@@ -450,10 +450,10 @@ def test_invalid_manual_data(self):
 
     def test_invalid_filepath(self):
         invalid_filepath = os.path.join(filepath, 'invalid_filepath_here')
-        with pytest.raises(NoMatchError):
+        with pytest.raises(ValueError, match='Did not find any files'):
```

Comment:
These should not have been `NoMatchError`, which is reserved for not finding a matching source (not not finding any files).